modules/orchestrator/orchestrator-core/src/main/java/org/apache/airavata/orchestrator/core/validator/impl/GroupResourceProfileValidator.java

```diff
@@ -0,0 +1,155 @@
+                                "for queue " + batchQueueResourcePolicy.getQueuename());
+                    } else {
+                        wallTimeResult.setResult(true);
+                        wallTimeResult.setErrorDetails("");
+                    }
+
+                    ValidatorResult nodeCountResult = new ValidatorResult();
+
+                    if (experimentNodeCount > batchQueueResourcePolicy.getMaxAllowedNodes()) {
+                        nodeCountResult.setResult(false);
+                        nodeCountResult.setErrorDetails("Job Execution node count " + experimentNodeCount +
+                                "exceeds the allowable node count for the group resource profile " + batchQueueResourcePolicy.getMaxAllowedNodes() +
+                                "for queue " + batchQueueResourcePolicy.getQueuename());
+                    } else {
+                        nodeCountResult.setResult(true);
+                        nodeCountResult.setErrorDetails("");
+                    }
+
+                    ValidatorResult cpuCountResult = new ValidatorResult();
+
+                    if (experimentCPUCount > batchQueueResourcePolicy.getMaxAllowedCores()) {
+                        cpuCountResult.setResult(false);
+                        cpuCountResult.setErrorDetails("Job Execution cpu count " + experimentCPUCount +
+                                "exceeds the allowable cpu count for the group resource profile" + batchQueueResourcePolicy.getMaxAllowedCores() +
+                                "for queue " + batchQueueResourcePolicy.getQueuename());
+                    } else {
+                        cpuCountResult.setResult(false);
+                        cpuCountResult.setErrorDetails("");
+                    }
+
+                    validatorResultList.add(wallTimeResult);
+                    validatorResultList.add(nodeCountResult);
+                    validatorResultList.add(cpuCountResult);
+                } else {
+                    ValidatorResult batchQueuePolicyResult = new ValidatorResult();
+                    logger.info("There is no batch queue resource policy specified for the group resource profile and queue name");
+                    batchQueuePolicyResult.setResult(true);
+                    validatorResultList.add(batchQueuePolicyResult);
+                }
+            } else {
+                queueNameResult.setResult(false);
+                queueNameResult.setErrorDetails("The specified queue " + queueName +
+                        " does not exist in the list of allowed queues for the group resource profile.");
+                validatorResultList.add(queueNameResult);
+            }
+        } else {
+            ValidatorResult result = new ValidatorResult();
+            logger.info("There is no compute resource policy specified for the group resource profile");
+            result.setResult(true);
```

Comment:
result should be false, but also, as mentioned above, we don't want to invalidate if there is no compute resource policy.

Reply:
As we discussed, I misread this, you're right the result should be true.